modules/machineset-yaml-aws.adoc

```diff
@@ -79,6 +125,21 @@ spec:
 ----
 $ oc get -o jsonpath='{.status.infrastructureName}{"\n"}' infrastructure cluster
 ----
+ifndef::infra[]
 <2> Specify the infrastructure ID, node label, and zone.
 <3> Specify the node label to add.
 <4> Specify a valid {op-system-first} AMI for your AWS zone for your {product-title} nodes.
+endif::infra[]
+ifdef::infra[]
+<2> Specify the infrastructure ID, `infra` node label, and zone.
+<3> Specify the `infra` node label.
+<4> Specify a taint to prevent user workloads from being scheduled on infra nodes.
+<5> Specify a valid {op-system-first} AMI for your AWS zone for your {product-title} nodes.
```

Comment:
If you have the OpenShift CLI installed, you can obtain the ami by running the following command:
```
$ oc -n openshift-machine-api \
    -o jsonpath='{.spec.template.spec.providerSpec.value.ami.id}{"\n"}' \
    get machineset/<infrastructureID>-worker-us-east-1a
```